yt_dlp/extractor/fuyintv.py

```diff
@@ -0,0 +1,27 @@
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class FuyinTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)fuyin\.(?:tv)/(?:html)/(?P<mov_id>\w+)/(?P<url_id>\d+)(?:\.html)'
+    _TESTS = [{
+        'url': 'https://www.fuyin.tv/html/2733/44129.html',
+        'info_dict': {
+            'id': '44129',
+            'ext': 'mp4',
+            'title': '第1集',
+        }
+    }]
+
+    def _real_extract(self, url):
+        playlist_id, video_id = self._match_valid_url(url).group('mov_id', 'url_id')
```

Comment:
`playlist_id` is unused?

Reply:
For now, it's unused, but i think can used later if set `--yes-playlist`

Reply:
Should i remove that ?

Reply:
--yes-playlist only applies when there is ambiguity whether a URL refers to video or playlist. It's not applicable here. If the group is unused, pls remove it

Reply:
Fixed in 3c495cbf546c7b489ebe597b34350d76c408d792